backend/api/Database/Models/Inspection.cs

```diff
@@ -128,4 +127,9 @@ public enum InspectionType
         ThermalVideo,
         Audio
     }
+    public enum AnalysisType
+    {
+        CarSeal,
+        RtjFlange
```

Comment:
Is this an analysis type? Should it not be gas leak?

Reply:
In eq_robot_utility_scripts it is named that way.